### ISSUE TITLE
Exception: Database error when running migration 

### ISSUE THREAD

**Issue by EmJotGeh** [pulsejet/memories#845]
**Describe the bug**

I updated my NC Instance to the new Version 27.1.2 the upgrade process stop with this error.

```Exception: Database error when running migration 505000Date20230821044807 for app memories
The column 'epoch' on table 'oc_memories' already exists.
Update failed
```

**To Reproduce**
Steps to reproduce the behavior: n.a.

**Screenshots**

**Platform:**

- OS: Ubuntu Server
- Browser: Firefox 118
- Memories Version: last 5.5.0
- Nextcloud Version: 27.1.2
- PHP Version: 8.1

**Additional context**

- Any errors in the JS console?
- Any errors in the Nextcloud server logs?

**Comment by TtuxX** [comment]
Same. As the error was blocking my instance with a mandatory `occ upgrade` that was systematically failing, I first tried deleting the `epoch` column, then went down the rabbit hole trying to solve MySQL errors 1-by-1, which surely was the wrong approach.
As it was not solving the issue and as other issues were arising, [I finally went the full reset way](https://memories.gallery/troubleshooting/#reset) and I am now running a clean `occ memories:index -f` task.
